src/XMLimport.cpp

```diff
@@ -1542,7 +1542,8 @@ int XMLimport::readActionGroup(TAction* pParent)
                 // or "2" (true) for backward compatibility
                 pT->mButtonState = (readElementText().toInt() == 2);
             } else if (name() == "buttonColor") {
-                pT->mButtonColor.setNamedColor(readElementText());
+                // Not longer present/used, skip over it if it is still in file:
+                skipCurrentElement();
```

Comment:
Scope creep... needs testing to ensure imports aren't broken, those have been a bit fragile

Reply:
It isn't creep IMHO - I removed the member because it wasn't used anywhere (so initialising it is pointless) in the current code and if we didn't make this change old files would cause Mudlet to moan about this entry.

Reply:
Please test this vigorously, any changes to export/import code are critical

Reply:
Well the current development code version will load a profile save from this code (and visa-versa) and neither case cause any loading errors to be reported in the debug output...